os/board/common/Kconfig

```diff
@@ -66,6 +66,12 @@ config ARCH_BOARD_HAVE_SECOND_FLASH
 if ARCH_BOARD_HAVE_SECOND_FLASH && (NFILE_DESCRIPTORS != 0)
 comment "Second Flash Partition Options"
 
+config FLASH_SECOND_START_ADDR
```

Comment:
Is this added to locate the ext flash address for downloading the rtk data binary? If yes, then we should check PR https://github.com/Samsung/TizenRT/pull/6303 first, after this, all binary integrate to resource.bin, and we do not need to consider the address offset anymore.

Reply:
Thank you for your review.

But this is not because of rtk binary.
This is to match the compatibility of the boards because the realtek downloader tool has a fixed ext flash start address of 0x0A000000.